src/pages/ws/ui/modal-tooltip.tsx

```diff
@@ -0,0 +1,23 @@
+import { ReactElement } from 'react'
+
+interface IModalTooltip {
+  sample: string
+  genotype: string
+  quality: number
+}
+
+export const ModalTooltip = ({
+  sample,
+  genotype,
+  quality,
```

Comment:
Since it's called ModalTooltip maybe you could make this component more generic? For example, it could receive a render function which renders anything you put there `<ModalTooltip render={() => (<div className="...">...</div)}/>

Reply:
May be just use `children` prop? `render` prop is relevant when it necessary to pass some arguments from component to render function

Reply:
Yes, it would be even better